### PR TITLE
retry loop.ko module loading (bsc #964046)

### PR DESCRIPTION
**Pull request by wfeldt**
Apparently it can happen that the kernel fails to load a module if it's
'too early'.

As a workaround, we retry a few times. This seems to help...

### CODE REVIEW

File: linuxrc.c [hunk 1]
```diff
@@ -1558,7 +1561,15 @@ void lxrc_add_parts()
...
         insmod_done = 1;
-        lxrc_run("/sbin/insmod /modules/loop.ko max_loop=64");
+        for(i = 0; i < MAX_LOOP_TRIES; i++) {
+          if(!lxrc_run("/sbin/insmod /modules/loop.ko max_loop=64")) break;
+          sleep(1);
+        }
+        if(i == MAX_LOOP_TRIES) {
+          log_show("Hm, module loading seems not to work, giving up.\n");
+          config.debugwait = 1;
+          LXRC_WAIT
```
Comment (mvidner) [inline linuxrc.c]: is a semicolon missing after LXRC_WAIT?

Comment (mvidner) [inline linuxrc.c]:
```c
#define LXRC_WAIT util_wait(__FILE__, __LINE__, __FUNCTION__);
```
So it isn't.

**Comment by mvidner** [comment]
LGTM.

**Comment by wfeldt** [comment]
It actually failed due to a glibc bug. It was fixed, so this patch is not required.

**Pull request closed without merging**